app/models/procedure_revision.rb

```diff
@@ -51,7 +51,7 @@ def add_type_de_champ(params)
 
     # the collection is orderd by (position, id), so we can use after_coordinate.position
     # if not present, a big number is used to ensure the element is at the tail
-    position = (after_coordinate&.position) || 100_000
+    position = params[:type_champ] == 'routage' ? -1 : (after_coordinate&.position || 100_000)
```

Comment:
pour garder le commentaire au bon endroit

```suggestion
    position = if params[:type_champ] == 'routage' 
      -1
     else
       # the collection is orderd by (position, id), so we can use after_coordinate.position
       # if not present, a big number is used to ensure the element is at the tail
       (after_coordinate&.position || 100_000)
     end
```